tests/test_data_quality.py

```diff
@@ -0,0 +1,77 @@
+import logging
```

Comment:
It would be nice to add a test for `additional_options` so we can test the partitions `pushdownPredicate` and increase our coverage

Reply:
Good idea, added